pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=61.2", "setuptools-scm"]
+requires = ["setuptools>=77.0", "setuptools-scm"]
```

As suggested in the [Python Packaging User Guide](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files):
```suggestion
requires = ["setuptools>=77.0.3", "setuptools-scm"]
```
I think 77.0.3 irons out bugs related to PEP 639 (77.0.0 and 77.0.2 have been yanked by the way).